### ISSUE TITLE
[Fabric 1.20.1] Parsing error loading recipes

### ISSUE THREAD

**Issue by xR4YM0ND** [Jeryn99/Gliders#61]
A lot of recipe parsing errors:

```
Parsing error loading recipe vc_gliders:diamond_glider_copper_upgrade
com.google.gson.JsonSyntaxException: Invalid or unsupported recipe type 'vc_gliders:glider_upgrading'
```

[latest.log](https://gist.github.com/xR4YM0ND/638a8ab69f8e40f795dfe88db2132aba)

**Comment by Jeryn99** [comment]
Interesting - Game still run and function as intended?

**Comment by Steveplays28** [comment]
In my experience the game still runs, but the recipes are broken and thus do not work.
It's weird because you do register that recipe type in the code.

**Comment by xR4YM0ND** [comment]
Any updates on this one?

**Comment by Jeryn99** [comment]
Resolved in 1.1.6
